### PR TITLE
Problem : Navigations doesn't look nice

### PR DESCRIPTION
**Pull request by MadanBhandari**
Solution : Side + Top Navigation should be always fixed and open side navigation inside the scope of top navigation

### CODE REVIEW

File: views/templates/normal/menu.html [hunk 1]
```diff
@@ -1,95 +1,63 @@
...
-<template name="menu">
-  BLOCKRAZOR<br />
-  {{#if currentUser}}
-  <div class="dropdown show">
-  <a class="btn btn-secondary dropdown-toggle" href="#" role="button" id="dropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-     {{currentUser.username}}
-  </a>
-
-  <div class="dropdown-menu" aria-labelledby="dropdownMenuLink">
-    <a class="dropdown-item" href="/profile">Edit Profile</a>
```
Comment (gazhayes) [inline views/templates/normal/menu.html]: Can users still edit their profile?

File: views/layouts/desktop.html [hunk 2]
```diff
@@ -1,33 +1,18 @@
...
         <!-- Content -->
         <div id="content">
-            {{> topnav}}
             {{> Template.dynamic template=main}}
             {{> Template.dynamic template=bottom}}
         </div>
     </div>
 </template>
-<template name="desktop2">
-  {{> sAlert}}
```
Comment (gazhayes) [inline views/layouts/desktop.html]: Are you sure this isn't needed?

File: views/templates/normal/menu.html (hunk 1 above)
Comment (MadanBhandari) [reply views/templates/normal/menu.html]: Now this is moved to top navigation, there is no profile section in side navigation.

File: views/layouts/desktop.html (hunk 2 above)
Comment (MadanBhandari) [reply views/layouts/desktop.html]: This was copy of old view, I made copy before editing but forgot to delete in first PR.

Comment (gazhayes) [reply views/layouts/desktop.html]: ok

**Pull request merged**